engine/core/src/main/java/org/pentaho/reporting/engine/classic/core/modules/output/table/xls/helper/ExcelPrinter.java

```diff
@@ -124,6 +124,7 @@ public void print( final LogicalPageKey logicalPageKey, final LogicalPageBox log
 
     if ( sheet == null ) {
       addNewSheet( logicalPage, contentProducer, excelTableContentProducer );
+      sheetRow = 0;
```

Comment:
please add a comment referring to the BUG causing this change, similar line 122.